core/includes/pager.inc

```diff
@@ -262,10 +264,10 @@ function theme_pager($variables) {
         'data' => $li_last,
       );
     }
-    return '<h2 class="element-invisible">' . t('Pages') . '</h2>' . theme('item_list', array(
+    return '<nav aria-label="Pagination>' . theme('item_list', array(
```

Comment:
I don't think we want the nav tag added here, or we'll get duplicated nav tags. All that needs to be changed on this line is removing the h2.

Reply:
It seems that there is no nav tag.
Here is the html I copied from the inspector
```
<nav aria-label="Pagination><div class=" item-list">
  <ul class="pager">
    <li class="pager-current odd first"><span aria-current="page">1</span></li>
    <li class="pager-item even"><a aria-label="last page" title="Go to page 2" href="/home?page=1">2</a></li>
    <li class="pager-next odd"><a aria-label="last page" title="Go to next page" href="/home?page=1">next ›</a></li>
    <li class="pager-last even last"><a aria-label="last page" title="Go to last page" href="/home?page=1">last »</a>
    </li>
  </ul>
</nav>
```